elements/bulbs-truncate-element/bulbs-truncate-element.scss

```diff
@@ -0,0 +1,27 @@
+.bulbs-truncate-element-parent-active {
```

Comment:
It works great. Maybe we can add a transition here? Would be a little nicer if we had a slight animation on the height of the active class or opacity attribute of the :after element. Maybe both?